background_tasks/thread_worker.py

```diff
@@ -0,0 +1,16 @@
+import queue
+
+from services.trigger_search import create_search_request
+
+job_queue = queue.Queue()
```

Comment:
are we able to make this a class/object instead with threading.Thread? right now it's just one static "job_queue" with a static function to process it. I think we should instantiate job queues and have start/stop and the queue as member fields (see comment in main.py)